### DISCUSSION TITLE
Color "blindness" misnomer

### DISCUSSION THREAD

**Discussion by jasonferrier** [community/community#16193] [Accessibility]
In your feature preview modal, one of the features is listed as color blindness.
Color vision deficiency is the proper term since persons that are affected are not blind to these colors.
https://www.aoa.org/healthy-eyes/eye-and-vision-conditions/color-vision-deficiency

**Comment by queenofcorgis** [comment]
Thanks @jasonferrier for letting us know and for teaching us something new, I've added a label to this so the proper team can easily find it :octocat:

**Comment by andrask** [comment]
I'm partially color blind and I don't mind naming it that. If I look for this feature, I use this term. And I like that Github has acted on this as the red vs green vs yellow balls just drive me crazy in other tools. :)

**Reply by adlai** [reply]
can you recommend any list of console tools that do not rely excessively on chromatic subtleties?

**Comment by Elegbedehhhhh** [comment]
How can I get access to trust wallet by address

**Comment by adlai** [comment]
I think the most powerful keyword would be **theming**, without any bias towards whether you suffer from any specific deficiencies. Maybe you're only coping with really bad lighting conditions... ultimately the symptoms are indistinguishable, and that is no reason, in my opinion, for the naming of such settings to hide behind verbose medical terminology. Obviously if some theme is specifically designed for deuteranopia, then that naming is logical.

**Comment by cehfisher** [comment]
Hi @jasonferrier - I wanted to follow up on this stale issue. Is this issue resolved this issue for you based on the [comment](https://github.com/orgs/community/discussions/16193#discussioncomment-2717204) from @queenofcorgis? If yes, could you please close the issue? If no, what additional changes are needed? Please supply detailed instructions to reproduce your issue and include screenshots and/or URLs, if applicable. Thank you! (cc @queenofcorgis)
